janusgraph-berkeleyje/src/test/java/org/janusgraph/graphdb/berkeleyje/BerkeleyGraphTest.java

```diff
@@ -112,19 +113,14 @@ public void testConcurrentConsistencyEnforcement() {
         //Do nothing TODO: Figure out why this is failing in BerkeleyDB!!
     }
 
-    @Test
+    @FlakyTest
```

Comment:
Is this test now suddenly flaky in GitHub Actions?

Reply:
The test hit a race condition on the github actions consistently.

Reply:
Do you have a link for the failure message? I wonder why it hits a race condition on GitHub Actions but not Travis.

Reply:
It seems that to be that GitHub Actions is a bit slower. You have to reduce the time of timeout which is not possible.

Reply:
I think this test is executed faster in GitHub Actions, thus it doesn't hit `timeout` as expected by the test. That is why it fails. I think we need to resolve this commit because it is now Disabled in this PR (which I think is right because the test is very flaky).